internal/core/data/v2/infrastructure/interfaces/db.go

```diff
@@ -18,6 +18,7 @@ type DBClient interface {
 	DeleteEventById(id string) errors.EdgeX
 	EventTotalCount() (uint32, errors.EdgeX)
 	EventCountByDevice(deviceName string) (uint32, errors.EdgeX)
+	DeletePushedEvents() errors.EdgeX
```

Comment:
This changes to `GetPushedEventReadingIds` and `DeleteEventsByIds` and `DeleteReadingsByIds`

Reply:
Updated.